src/main/java/org/traccar/protocol/Tlt2hProtocol.java

```diff
@@ -41,6 +41,7 @@ protected void addProtocolHandlers(PipelineBuilder pipeline, Config config) {
         addServer(new TrackerServer(config, getName(), true) {
             @Override
             protected void addProtocolHandlers(PipelineBuilder pipeline, Config config) {
+                pipeline.addLast(new CharacterDelimiterFrameDecoder(32 * 1024, "##\r\n"));
```

Comment:
UDP protocols shouldn't have any frame decoders. I think there right solution is to handle it in the protocol decoder.

Reply:
Okay, I added a check for empty messages in the protocol decoder. Could you please check again?